gcp/modules/cert-manager/resources/issuers.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: letsencrypt-staging
+  namespace: gpii
```

Comment:
Sergey thinks it is a quick fix to have one Issuer per cluster (rather than being tucked into the gpii namespace). If it is, he will fix it.

Sergey, if you choose not to fix this, please add a comment explaining why this is hardcoded into the gpii namespace.

Reply:
This is fixed by moving these resources to kube-system and changing issuer kind to ClusterIssuer.